support-frontend/conf/routes

```diff
@@ -91,16 +91,13 @@ GET  /$country<(uk|us|au|eu|int|nz|ca)>/subscribe                  controllers.S
 # subscribe route. We just redirect to the subscriptions site and let its geolocation handle it.
 GET  /:country/subscribe                                           controllers.SubscriptionsController.legacyRedirect(country: String)
 
-GET  /digital                                                      controllers.DigitalSubscriptionController.digitalGeoRedirect(orderIsAGift: Boolean = false)
+# Digital subscriptions
```

Comment:
top digital route removed?

Reply:
Nice spot, thanks. I think we should be trying to keep things as close to current as possible, so I've added it back in.